libass/ass_font.c

```diff
@@ -474,7 +474,7 @@ int ass_font_get_index(ASS_FontSelector *fontsel, ASS_Font *font,
         int face_idx;
         ass_msg(font->library, MSGL_INFO,
                 "Glyph 0x%X not found, selecting one more "
-                "font for (%s, %d, %d)", symbol, font->desc.family,
+                "font for (%.*s, %d, %d)", symbol, (int)font->desc.family.len, font->desc.family.str,
```

Comment:
It's not a bug (and cannot crash) as `ASS_Font`'s `family` is always zero-terminated (and used as such [here](https://github.com/libass/libass/blob/master/libass/ass_fontselect.c#L696), for example). Commit message should be changed accordingly. Also space after `(int)`.

Reply:
No, it's a bug, because it tries to print `font->desc.family` instead of `font->desc.family.str`.

Reply:
Added spaces after `(int)`, otherwise unchanged.
Imho it's better to use `%.*s` here — even if `%s` also works — to foster consistency and avoid having to wonder if/why the currently used `ASS_StringView` is zero-terminated. At those places, where zero-termination for `ASS_StringView` is known to exist and used, we currently are and imho should be explicit about it, see here
https://github.com/libass/libass/blob/b9f34688f6ef1988136ba5e90a91ae8ac9b8a55f/libass/ass_fontselect.c#L696
and here
https://github.com/libass/libass/blob/b9f34688f6ef1988136ba5e90a91ae8ac9b8a55f/libass/ass_render.c#L1185